src/helpers.ts

```diff
@@ -34,18 +34,13 @@ function _getRepository<T extends IEntity>(
   docId?: string,
   subColName?: string
 ): BaseFirestoreRepository<T> {
-  const {
-    firestoreRef,
-    getRepository,
-    getSubCollection,
-    getCollection,
-  } = getMetadataStorage();
-
-  if (!firestoreRef) {
+  const metadataStorage = getMetadataStorage();
+
+  if (!metadataStorage.firestoreRef) {
     throw new Error('Firestore must be initialized first');
   }
 
-  const repository = getRepository(entity);
+  const repository = metadataStorage.getRepository(entity);
```

Comment:
I don't think that this change is inside the scope of this pull request (updating the case of the functions exported by fireorm).

Reply:
The reason I did this is because it was conflicting with `getRespository` in the upper scope 😄

Reply:
Oh, you're totally right 😄. I welcome this change then. Make sense to rename any of the functions?

Reply:
I think the naming of the functions is okay as it's for internal use only and it's clearly linked to the class